### PR TITLE
198-banner-component-converter

### PR DESCRIPTION
**Pull request by sharanagoudapatil**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://198-banner-component-converter--danaher-ls-aem--hlxsites.hlx.page

### CODE REVIEW

**Review by mhaack — APPROVED** [review]

File: tools/actions/convert/test/fixtures/product-category-semantic.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          <div>
+            <ul>
+              <li><a href="/us/en/products.html">Products</a></li>
+              <li><a href="/us/en/products/capillary-electrophoresis-systems.html">Capillary Electrophoresis Instrument</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <hr>
+      <div class="banner">
```
Comment (mhaack) [inline tools/actions/convert/test/fixtures/product-category-semantic.html:18]: The banner is full with, same like the breadcrumb, I think it should be in the first section aka. before the <hr> tag.

File: tools/actions/convert/test/fixtures/blog-hub-semantic.html [hunk 2]
```diff
@@ -13,6 +13,13 @@
         </div>
       </div>
       <hr>
+      <div class="banner">
```
Comment (mhaack) [inline tools/actions/convert/test/fixtures/blog-hub-semantic.html:16]: The banner is full with, same like the breadcrumb, I think it should be in the first section aka. before the <hr> tag.

**Review by davenichols-DHLS — APPROVED** [review]
Looks OK code wise, @mhaack  - can you please approve and we can merge?

File: tools/actions/convert/test/fixtures/product-category-semantic.html (hunk 1 above)
Comment (sharanagoudapatil) [reply tools/actions/convert/test/fixtures/product-category-semantic.html:18]: @mhaack - Sorry, if I move it to before <hr> again tests failing.

**Pull request merged**